### PR TITLE
Add docker integration

### PR DESCRIPTION
**Pull request by neurodrone**
- This should start ceph exporter on `9190` port as a default. It can be changed later if need be.

cc: @nickvanw @mdlayher @cagedmantis

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ENV GOROOT /goroot
+ENV GOPATH /go
+ENV PATH $GOROOT/bin:$PATH
+ENV APPLOC $GOPATH/src/github.com/digitalocean/ceph_exporter
+
+ADD . $APPLOC
+
+RUN apt-get update && \
+    apt-get install -y build-essential git curl && \
+    apt-get install -y librados-dev librbd-dev
```
Comment (mdlayher) [inline Dockerfile]: Any reason these two can't be part of the first `apt-get install` command?

**Comment by mdlayher** [comment]
:+1: pending comment, though I am no Docker expert so CC: @nickvanw and @cagedmantis.

File: Dockerfile (hunk 1 above)
Comment (neurodrone) [inline Dockerfile]: No other reason than helping separate concerns/readability especially if we only need to modify or move the ceph/rados/rbd installation part without a performance penalty.

Comment (mdlayher) [inline Dockerfile]: Sounds good to me.

**Comment by cagedmantis** [comment]
Also not a Docker expert but this LGTM

**Pull request merged**